graylog2-web-interface/src/components/sidecars/configuration-forms/CollectorForm.jsx

```diff
@@ -135,55 +155,58 @@ const CollectorForm = createReactClass({
                       onChange={this._formDataUpdate('service_type')}
                       placeholder="Service Type"
                       required />
-              <HelpBlock>Choose the service type this collector is meant for.</HelpBlock>
+              <HelpBlock>{this._formatValidationMessage('service_type', 'Choose the service type this collector is meant for.')}</HelpBlock>
             </FormGroup>
 
-            <FormGroup controlId="operating_system">
+            <FormGroup controlId="operating_system"
+                       validationState={this._validationState('node_operating_system')}>
               <ControlLabel>Operating System</ControlLabel>
               <Select inputProps={{ id: 'node_operating_system' }}
                       options={this._formatOperatingSystems()}
                       value={this.state.formData.node_operating_system}
                       onChange={this._formDataUpdate('node_operating_system')}
                       placeholder="Name"
                       required />
-              <HelpBlock>Choose the operating system this collector is meant for.</HelpBlock>
+              <HelpBlock>{this._formatValidationMessage('node_operating_system', 'Choose the operating system this collector is meant for.')}</HelpBlock>
             </FormGroup>
 
             <Input type="text"
                    id="executablePath"
                    label="Executable Path"
                    onChange={this._onInputChange('executable_path')}
-                   help="Path to the collector executable"
-                   value={this.state.formData.executable_path}
+                   bsStyle={this._validationState('executable_path')}
+                   help={this._formatValidationMessage('executable_path', 'Path to the collector executable')}
+                   value={this.state.formData.executable_path || ''}
                    required />
 
             <Input type="text"
                    id="executeParameters"
                    label="Execute Parameters"
                    onChange={this._onInputChange('execute_parameters')}
                    help={<span>Parameters the collector is started with.<strong> %s will be replaced by the path to the configuration file.</strong></span>}
-                   value={executeParameters} />
+                   value={executeParameters || ''} />
 
             <Input type="text"
                    id="validationParameters"
                    label="Parameters for Configuration Validation"
                    onChange={this._onInputChange('validation_parameters')}
                    help={<span>Parameters that validate the configuration file. <strong> %s will be replaced by the path to the configuration file.</strong></span>}
-                   value={validationParameters} />
+                   value={validationParameters || ''} />
 
             <FormGroup controlId="defaultTemplate">
               <ControlLabel>Default Template</ControlLabel>
```

Comment:
The default template field does not show any validation state, but it is currently required.

I have found the same that @mpfz0r did, when filling the collector form, the create button is disabled until I fill the default template. From the network requests I can see that the backend returns:

```JSON
{"errors":{"default_template":["Collector default template cannot be empty."]},"failed":true}
```